zaza/model.py

```diff
@@ -790,14 +801,33 @@ def f(x):
         filesystem_store_datadir = /var/lib/glance/images/
         default_store = file
 
-    use:
+    Call the check via its sync wrapper::
 
         expected_contents = {
             'DEFAULT': {
                 'debug': ['False']},
             'glance_store': {
                 'filesystem_store_datadir': ['/var/lib/glance/images/'],
                  'default_store': ['file']}}
+
+        block_until_oslo_config_entries_match(
+            'modelname',
+            'glance',
+            '/etc/glance/glance-api.conf',
+            expected_contents)
+
+    :param model_name: Name of model to query.
+    :type model_name: str
+    :param application_name: Name of application
+    :type application_name: str
+    :param remote_file: Remote path of file to transfer
+    :type remote_file: str
+    :param expected_contents: The key values pairs in their corresponding
```

Comment:
value (singluar), I believe the plurality of pairs speaks for both key and value